### PR TITLE
Website update graphics and mobile

### PR DESCRIPTION
**Pull request by lindaicing**

### CODE REVIEW

**Comment by lindaicing** [comment]
New Homepage Content:
Screen Shot 2020-10-02 at 12 25 14 AM
Screen Shot 2020-10-02 at 12 25 48 AM



Info page main:
Screen Shot 2020-10-02 at 12 24 08 AM
Screen Shot 2020-10-02 at 12 24 48 AM

File: Website-and-Backend/src/main/resources/templates/index.html [hunk 1]
```diff
@@ -94,19 +94,20 @@ <h2><span class="colorLGreen">How to use</span> <span class="colorCream">Ethicli
...
+                    <p>Sometimes while shopping, we'll recomend more ethical products based on items that you are viewing.</p>
                 </article>
             </div>
             <div class="threes">
                 <article class="imageDesc">
                     <h3>About Ratings</h3>
-                    <p>We use a data-driven algorithm based on companies' certifications and partnerships to score online shops,
-                        saving you the time and energy it takes to be sure your purchase reflects your values.</p>
+                    <p>We use a data-driven algorithm based on companies' certifications and partnerships to score online shops. We aim to
+                        save you the time and energy it takes to be sure your purchase reflects your values.</p>
```
Comment (CubeOfCheese) [inline Website-and-Backend/src/main/resources/templates/index.html:110]: I feel like this sounds better the way it was before

**Comment by CubeOfCheese** [comment]
Can you update the tagline?
"Ethicli helps you choose better products" ->
"Ethicli helps you shop for ethical products"

**Comment by CubeOfCheese** [comment]
Good job with the updated info page for mobile! :)

File: Website-and-Backend/src/main/resources/templates/index.html [hunk 2]
```diff
@@ -94,12 +94,13 @@ <h2><span class="colorLGreen">How to use</span> <span class="colorCream">Ethicli
...
                 <article class="gallery halfWidth">
                     <img src="images/extension-error.png" alt="Full Extension">
-                    <img src="images/extension-bluesign.png" alt="Full Extension">
+                    <img src="images/extension.png" alt="Full Extension">
                 </article>
                 <article class="imageDesc">
                     <h3>How it Works</h3>
                     <p>While shopping, click on the Ethicli icon in your browser menu to see a breakdown of a shop’s score.
                         Want to know more? Select “View Details” to access links, sources, and suggestions for better products!</p>
+                    <p>Sometimes while shopping, we'll recomend more ethical products based on items that you are viewing.</p>
```
Comment (CubeOfCheese) [inline Website-and-Backend/src/main/resources/templates/index.html:103]:
```suggestion
                    <p>Sometimes while shopping, we'll recommend more ethical products based on what you're shopping for.</p>
```
recommend has a typo
"items that you are viewing" doesn't sound natural

**Review by CubeOfCheese — CHANGES_REQUESTED** [review]
Sorry I didn't notice this the first time around

**Review by CubeOfCheese — APPROVED** [review]
:)

**Pull request merged**